libraries/Microsoft.Bot.Builder.LanguageGeneration/Diagnostic.cs

```diff
@@ -88,15 +91,7 @@ public class Diagnostic
 
         public override string ToString()
         {
-            // ignore error range if source is "inline"
-            if (Source == "inline")
-            {
-                return $"[{Severity}] {Message}";
-            }
-            else
-            {
-                return $"[{Severity}] {Range}: {Message}";
-            }
+            return $"[{Severity}] {Range}: {Message}";
```

Comment:
I'm not sure should we include "source" here, my first reaction is yes, but i haven't think about all scenarios